lib/xlsx_reader/parsers/worksheet_parser.ex

```diff
@@ -247,15 +247,23 @@ defmodule XlsxReader.Parsers.WorksheetParser do
 
   defp emit_row(state) do
     state = handle_omitted_rows(state)
+    row = state.row |> sanitize_row() |> Enum.reverse()
 
     %{
       state
       | row: nil,
-        rows: [Enum.reverse(state.row) | state.rows],
+        rows: [row | state.rows],
         expected_row: state.current_row + 1
     }
   end
 
+  defp sanitize_row(row) do
+    Enum.map(row, fn
+      :expect_chars -> ""
```

Comment:
At first I've thought about making them `nil`, but as the other cases are handled as `""`, also returning empty for this case.